### DISCUSSION TITLE
Problem to run a script in remote mode

### DISCUSSION THREAD

**Discussion by Remi1945** [Stellarium/stellarium#2354] [Q&A]
Hello
I am trying to run the double_stars script with curl as shown in the documentation.
But when I type **curl -d 'id=double_stars.ssc' http://localhost:8090/api/scripts/run**
_The answer is need parameter id_
What is wrong?

**Comment by gzotti** [comment]
Works for me on Stellarium running on Win10 and curl on a WSL console. Disclosing your operating system may help troubleshooting.

**Reply by Remi1945** [reply]
Sorry I my OS is Windows10 but I don't use a WSL console.
